README.md

```diff
@@ -24,6 +24,16 @@ conda create --name torcharrow python=3.7
 conda activate torcharrow
 ```
 
+### Version Compatibility
+
+The following is the corresponding `torcharrow` versions and supported Python versions.
+
+| `torch`            | `torcharrow`        | `python`          |
+| ------------------ | ------------------ | ----------------- |
+| `main` / `nightly` | `main` / `nightly` | `>=3.7`, `<=3.10` |
+| `1.12.0` /  `1.13.0`| `0.1.0` / `0.2.0` | `>=3.7`, `<=3.10` |
```

Comment:
Does the version after `/` refer to nightly?
If it refers to two different official releases, you should split it into two rows.

Reply:
yeah that's how I understood it. 0.1.0 is the official release, and 0.2.0 is the next version (nightly, not official yet).

Reply:
Technically, the line 33 refers to nightly release.

Reply:
I see ... then do you need to have the matrix for official release at all? Or just the main / nightly?